service/src/main/java/bio/terra/pipelines/dependencies/stairway/StairwayJobMapKeys.java

```diff
@@ -19,9 +22,4 @@ public enum StairwayJobMapKeys {
   public String getKeyName() {
     return keyName;
   }
-
-  public static boolean isRequiredKey(String keyName) {
```

Comment:
is this something that isnt useful to have?  I can imagine a place where we want to get a list of required keys.  Was it not being used before?

Reply:
ugh, agreed, but the place I wanted to use it (when setting up a new flight) is written in a way that wasn't obvious how to use this. but I'm going to take another look, because this is a much clearer way to specify the required keys than what we have.